_posts/2015-03-23-aerogear-ups-103.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: news
+section: news
+title: AeroGear UnifiedPush Server 1.0.3 released
+author: matzew
+module: push
+releases: 1.0.3
+---
+
+The AeroGear team is happy to announce the version 1.0.3 of the UnifiedPush Server! The release focuses on stabilization and adds a few new features
+
+* Migration support for 1.0.x users as explained in our [user guide](https://aerogear.org/docs/unifiedpush/ups_userguide/index/#migration-guide)
```

Comment:
it is optional. if you want to upgrade, see link for details

On Monday, March 23, 2015, Lukáš Fryč <notifications@github.com> wrote:

> In _posts/2015-03-23-aerogear-ups-103.md
> <https://github.com/aerogear/aerogear.org/pull/566#discussion_r26924747>:
>
> > @@ -0,0 +1,25 @@
> > +---
> > +layout: news
> > +section: news
> > +title: AeroGear UnifiedPush Server 1.0.3 released
> > +author: matzew
> > +module: push
> > +releases: 1.0.3
> > +---
> > +
> > +The AeroGear team is happy to announce the version 1.0.3 of the UnifiedPush Server! The release focuses on stabilization and adds a few new features
> > +
> > +* Migration support for 1.0.x users as explained in our [user guide](https://aerogear.org/docs/unifiedpush/ups_userguide/index/#migration-guide)
>
> Is migration a feature?
>
> If yes, I would start with the features that caused a need for migration
> and then explained they unfortunately requires database migration.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear.org/pull/566/files#r26924747>.
>


-- 
Sent from Gmail Mobile